templates.go

```diff
@@ -151,35 +165,37 @@ func getTemplates() *template.Template {
 			})();
 		}
 	</script>
-	<footer>
-	{{ if eq .Footer "-" }}
-	{{ else if eq .Footer ""}}
-	Secured with <a href="https://github.com/oauth2-proxy/oauth2-proxy#oauth2_proxy">OAuth2 Proxy</a> version {{.Version}}
-	{{ else }}
-	{{.Footer}}
 	{{ end }}
-	</footer>
+` + footer + `
+{{end}}`)
+	if err != nil {
+		logger.Fatalf("failed parsing template %s", err)
+	}
+
+	t, err = t.Parse(`{{define "error.html"}}` + header + `
```

Comment:
I wonder if we should extend the template to take the button link and the button message and use a single error template, WDYT?